tests/testthat/test-tait-bryan.R

```diff
@@ -1,11 +1,19 @@
+test_that("unspecified unit argument throws error", {
+  expect_error(tait_bryan(yaw = 0, pitch = 0, roll - 0))
+})
+
 test_that("null rotations have no effect.", {
 
   set_dddr_semantics(
     axes = semantics_axes_unity,
     angles = semantics_angles(intrinsic = "ypr", hand = "left")
   )
   expect_equal(
-    tait_bryan(yaw = 0, pitch = 0, roll = 0),
+    tait_bryan(yaw = 0, pitch = 0, roll = 0, unit="radians"),
+    quat(1, 0, 0, 0)
+  )
+  expect_equal(
```

Comment:
Instead of copy-and-pasting the test for both radians and degrees, I think the philosophy here is to make a separate test for degrees with handcoded values and also some degrees-to-radians equivalency. It feels like the tests with duplicate values aren't 'testing one thing'.